From: Ben Hutchings <ben@decadent.org.uk>
Subject: [PATCH] keyboard: Export k_handler again

Commit 97f5f0cd8cd0a05449cbb77d1e6f02e026875802 "Input: implement
SysRq as a separate input handler" removed the export of k_handler.
However, speakup currently depends on this.

--- a/drivers/char/keyboard.c
+++ b/drivers/char/keyboard.c
@@ -81,7 +82,8 @@
 typedef void (k_handler_fn)(struct vc_data *vc, unsigned char value,
 			    char up_flag);
 static k_handler_fn K_HANDLERS;
+k_handler_fn *k_handler[16] = { K_HANDLERS };
+EXPORT_SYMBOL_GPL(k_handler);
-static k_handler_fn *k_handler[16] = { K_HANDLERS };
 
 #define FN_HANDLERS\
 	fn_null,	fn_enter,	fn_show_ptregs,	fn_show_mem,\
